<?xml version="1.0" encoding="UTF-8"?>
<!-- Vị trí: /ui/widgets/settings/AppearanceSettingsWidget.ui -->
<!-- Phiên bản: 1.6 (Sửa lỗi Build và cập nhật theo kế hoạch) -->
<ui version="4.0">
 <class>AppearanceSettingsWidget</class>
 <widget class="QWidget" name="AppearanceSettingsWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Giao diện (Theme)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="lightThemeRadioButton">
        <property name="text">
         <string>Sáng (Light)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="darkThemeRadioButton">
        <property name="text">
         <string>Tối (Dark)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="systemThemeRadioButton">
        <property name="text">
         <string>Theo hệ thống</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Phông chữ (Font)</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Phông chữ hiện tại:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="fontPreviewLabel">
        <property name="text">
         <string>System Font, 9pt</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="changeFontButton">
        <property name="text">
         <string>Thay đổi Phông chữ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
</ui>
